server/Internal/Server/Server.cs

```diff
@@ -0,0 +1,168 @@
+namespace Greet.Server {
+
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Runtime.CompilerServices;
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.OpenApi.Models;
+    using Microsoft.Extensions.Diagnostics.HealthChecks;
+    using Serilog;
+    using Serilog.Context;
+    using Serilog.Core;
+    using Serilog.Events;
+    using Serilog.Templates;
+    using Grpc.Core;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+    
+    using Greet;
+    using Greet.Services;
+    using Greet.Server;
+    using MiddlewareListInterceptors;
+    using LogAttrs;
+
+    class RemovePropertiesEnricher : ILogEventEnricher
+    {
+        public void Enrich(LogEvent le, ILogEventPropertyFactory lepf)
+        {
+            le.RemovePropertyIfPresent("SourceContext");
+            le.RemovePropertyIfPresent("RequestId");
+            le.RemovePropertyIfPresent("RequestPath");
+            le.RemovePropertyIfPresent("ConnectionId");
+        }
+    }
+
+    public static class LoggerExtensions
+    {
+        public static ILogger WithCtx(this ILogger logger, ServerCallContext context,
+            [CallerFilePath] string callerFilePath = "",
```

Comment:
Ideally, we want the location information to be generated in the middleware. User shouldn't have to worry about this

Reply:
moved withCtx to CtxLogger in the middleware